### PR TITLE
Adding core.xhr from freedom. Runs the integration tests too

### PR DESCRIPTION
**Pull request by ryscheng**
Pretty small change. But figured I'd pull request since I'll cut a new release here

### CODE REVIEW

**Comment by ryscheng** [comment]
All tests pass btw

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "es5-shim": "^4.0.0",
     "es6-promise": "^2.0.0",
     "folderify": "^0.6.0",
-    "freedom": "~0.6.14",
+    "freedom": "~0.6.16",
```
Comment (bemasc) [inline package.json]: 16?

Comment (bemasc) [inline package.json]: Oh, yep, 16 indeed.

**Comment by bemasc** [comment]
:+1:

**Pull request merged**